### ISSUE TITLE
Set record status via pac power-fx

### ISSUE THREAD

**Issue by lar-mar** [microsoft/Power-Fx#2507]
We are currently using PowerShell scripts and the corresponding PS Modules to connect to Dataverse to set the status of specified records. e.g. SLA

When updating a SLA via Solution import you have to ensure it's activated afterwards again.

However, I now wanted to replace the PS usage with pac cli as it is available in Azure Pipelines without additional effort and Power-Fx is maybe easier to use.

When I want to update the status of record like SLA or process, I'm facing an error message:
![image](https://github.com/microsoft/powerplatform-build-tools/assets/15412588/23edda20-9633-49c1-b078-59d2edad25c0)

![image](https://github.com/microsoft/powerplatform-build-tools/assets/15412588/98db3c4e-54d3-412a-b09a-e0d354fb10e8)

When I use the exactly same command within a Canvas app it works fine, so it seems it is a limitation of pac power-fx.
Are there any plans to close this gap?

**Comment by anderson-joyle** [comment]
We are tracking this internally. There is no hard estimate but we are interested in fixing this soon.

**Comment by anderson-joyle** [comment]
For internal reference: https://github.com/microsoft/Power-Fx-Dataverse/issues/480

**Comment by bennyboy23** [comment]
This seems to be working now for me, i used it for activating SLAs
`Patch(
    SLAs,
    LookUp(
        SLAs,
        Name = "testSLA"
    ),
    {
        'Status Reason': 'Status Reason (SLAs)'.Active,
        Status:'Status (SLAs)'.Active,
        'Is Default': 'Is Default (SLAs)'.Yes
    }
)`
Ran this with `pac power-fx run`

**Comment by lar-mar** [comment]
@bennyboy23 
You are right. I forgot to update this issue. I already use pac for months to set the status of SLAs in my pipelines.
